app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide :title, t("about") %>
+<h1><%= t("about") %></h1>
+<p>
+  The <%= link_to t("book_title"), "http://www.railstutorial.org/" %> is a
```

Comment:
Sử dụng I18n cho toàn bộ file đi em

Reply:
Chỗ này bạn nên cho vào 1 text trong I18n là được nhé. Ví dụ
book_title: "the %{link1} is a %{link2} and %{link3} to teach web development with %{link4} "
Sau đó khi gọi I18n bạn chỉ cần truyền vào link 1, link 2 ,link3 là được nhé 
Fix tương tự ở những trường hợp khác nhé

Reply:
làm theo cách này nó hiển thị plain text anh